### DISCUSSION TITLE
WebApi.Hosting.AzureFunctions still relevant ?

### DISCUSSION THREAD

**Discussion by fgheysels** [arcus-azure/arcus.webapi#464] [General]
How relevant is WebApi.Hosting.AzureFunctions ?
It contains middleware to enforce JSON content in requests to functions.   Why do we want to enforce that ?

**Comment by stijnmoreels** [comment]
Bc Azure Functions didn't do that in the past, unlike Web API.
https://www.codit.eu/blog/more-secure-dto-deserialization-in-azure-functions-applications-in-three-steps/
Not sure what has been developed by Microsoft since.
